theequityfund/the-equity-fund-wp-theme
# PHP dependency upgrades

## Changes

- Updates composer config to require php version 8.1 (this matches the production and staging environments).
- Upgrades to Timber version 2.3.0.
- Adds latest versions of required WordPress plugins to `composer.json`.
- Reinstalls frontend dependencies to fix a missing package dependency issue.

## How To Test

1. Pull down this branch. start the environment with `./bin/start
2. Confirm that the latest versions of plugins are installed.
3. Confirm that the site works as expected.
